src/main/scala/db/migration/V6__Populate_Subject_Id_column.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Part of NDLA article_api.
+ * Copyright (C) 2016 NDLA
+ *
+ * See LICENSE
+ */
+
+package db.migration
+
+import java.sql.Connection
+import org.flywaydb.core.api.migration.jdbc.JdbcMigration
+import scalikejdbc._
+import no.ndla.articleapi.ComponentRegistry.migrationApiClient
+
+class V6__Populate_Subject_Id_column extends JdbcMigration {
```

Comment:
Er kanskje lettest å gjøre det slik du har gjort det, med å splitte opp `V5__Create_External_Subject_Id_Column.sql` og i `V6__Populate_Subject_Id_column.scala`. Hadde allikevel helst sett at vi ikke bumpet to versjoner i version på databasen for denne endringen.